### DISCUSSION TITLE
Need help for mapping Modifier flags + key

### DISCUSSION THREAD

**Discussion by leminhthanh142** [pqrs-org/Karabiner-Elements#3470] [Q&A]
Hi, i have this configuration to mapping left_control + left_arrow to left_control + button5 for switch desktop, i know that it already have a complex configuration for button4 and button5 but i want to combine with other keys so i can macro it for more functions
```
{
  "title": "",
  "rules": [
    {
      "description": "left_ctrl + left_arrow to left_ctrl + button5",
      "manipulators": [
        {
          "type": "basic",
          "from": {
            "modifiers": {
              "mandatory": [
                "left_control"
              ]
            },
            "pointing_button": "button5"
          },
          "to": [
            {
              "repeat": false,
              "key_code": "left_arrow",
              "modifiers": [
                "left_control"
              ]
            }
          ]
        }
      ]
    }
  ]
}
```
the configuration file was generated using [Karabiner Complex Modification](https://genesy.github.io/karabiner-complex-rules-generator/)
i have try to reverse the order but got no luck
Anyone have try this?

**Comment by evan-liu** [comment]
Try: 

```json
[
  {
    "type": "basic",
    "from": {
      "key_code": "left_arrow",
      "modifiers": {
        "mandatory": [
          "left_control"
        ]
      }
    },
    "to": [
      {
        "pointing_button": "button5",
        "modifiers": [
          "left_control"
        ]
      }
    ]
  }
]
```

(Generated by [karabiner.ts](https://github.com/evan-liu/karabiner.ts), 
`map('←', 'left_control').toPointingButton('button5', 'left_control')` )
